include/emp/bits/BitVector.hpp

```diff
@@ -701,6 +700,72 @@ namespace emp {
       return (int) (pos_found + (field_id * FIELD_BITS));
     }
 
+    /// Pop the last bit in the vector
+    bool PopBack() {
+      bool val = Get(num_bits-1);
+      if (val)  Set(num_bits-1, 0);
```

Comment:
I would remove the if statement. If the value is already zero, we can just set it to zero again no problem